starter_code/app.js

```diff
@@ -1,18 +1,46 @@
-
 const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
+const app = express();
+
+const hbs = require('hbs');
+const path = require('path');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const punkAPI = new PunkAPIWrapper();
 
-app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+app.set('views', __dirname + '/views');
+app.set('view engine', 'hbs');
+app.set('layout', __dirname + '/views/layout.hbs');
+hbs.registerPartials(__dirname + '/views/partials');
+
+// Route
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+// ---------------
 
-app.listen(3000);
\ No newline at end of file
+app.get('/beers', (request, response, next) => {
+  punkAPI
+    .getBeers()
+    .then(beers => {
+      response.locals.beers = beers;
+      response.render('beers.hbs');
+      console.log(beers);
+    })
+    .catch(error => {
+      console.log(error);
+    });
+});
+
+// ---------------
+
+app.get('/random-beers', (request, response, next) => {
+  response.render('beers.hbs');
```

Comment:
This is missing the logic of fetching the random beer and passing it as data to the view.